frontend/views/containers/proposals/Mincome.vue

```diff
@@ -1,20 +1,118 @@
-<template lang="pug">
-proposal-template(
-  :subtitle="L('Change minimum income')"
-  :onsubmit='handleSubmit'
-)
-  | I step to change minimum income it&apos;s on the way.
+<template lang='pug'>
+  proposal-template(
+    title= 'L("Change minimum income")'
+    footer='L("According to your voting rules, 8 out of 10 members will have to agree with this.")'
```

Comment:
ProposalTemplate already does i18n for us when receiving the title and footer, so we don't need to use `L()` here.